390/390_Nidhi_Vinod.Rmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "390/Nidhi_Vinod"
+author: "Nidhi"
+date: "9/16/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+```{r}
+install.packages("tidyverse")
```

Comment:
Hey Nidhi, here is a code review! 

First off, let's delete this install.packages line. If someone else runs this code, you'd be reinstalling tidyverse for them without their permission!